src/batcher/op-batcher/op_batcher_launcher.star

```diff
@@ -87,7 +90,10 @@ def get_batcher_config(
         "--max-channel-duration=1",
         "--l1-eth-rpc=" + l1_config_env_vars["L1_RPC_URL"],
         "--private-key=" + gs_batcher_private_key,
-        "--data-availability-type=blobs",
+        "--altda.enabled=" + str(da_server_context.enabled),
```

Comment:
Could you add a check for if the da_server_context exists or not before using this? If you have a kurtosis config without `da_server` in the `additional_services`, then the context won't exist and the run will fail. The easiest way to do this is to have a `da_server_enabled` field, set that to false by default and check for truth and append the values if true. E.g how we do this in a different package: https://github.com/ethpandaops/ethereum-package/blob/main/src/cl/teku/teku_launcher.star#L237

Reply:
That cannot happen, because I populate those fields first with default values here https://github.com/ethpandaops/optimism-package/blob/a6c4860104afdf48369f09bf536861dbe68cfeb0/src/l2.star#L53
and the disabled (default) context will populated enabled with false
https://github.com/ethpandaops/optimism-package/blob/a6c4860104afdf48369f09bf536861dbe68cfeb0/src/alt-da/da-server/da_server_launcher.star#L93

But if it's confusing I can refactor to use the same method that you guys use.

Reply:
Nvm, found the bug! [3aaf3ac](https://github.com/ethpandaops/optimism-package/pull/69/commits/3aaf3acd1e0149f6b04fcaadff8827ed8150095a)